.travis.yml

```diff
@@ -13,6 +13,7 @@ matrix:
 
 install:
   # Install dependencies
+  - pip install --upgrade setuptools
```

Comment:
By default, Travis's Ubuntu VM uses setuptools 12.0, which doesn't support conditional dependencies without jumping through hoops (see [Conditional Python Dependencies](https://hynek.me/articles/conditional-python-dependencies/)). To keep things simple, I upgraded setuptools to conditionally install the backported `pathlib` package for 2.7 and 3.3. It's included in 3.4+.

Since this is my first time creating a public PyPI package, any suggestions on best practices would be appreciated.